net.damonlynch.rapid_photo_downloader.yaml

```diff
@@ -0,0 +1,153 @@
+app-id: net.damonlynch.rapid_photo_downloader
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-23.08
+cleanup:
+  - "/include"
+  - /doc
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /man
+  - /share/doc
+  - /share/gtk-doc
+  - /share/man
+  - /share/vala
+  - "*.a"
+  - "*.la"
+  - "/lib/*.a"
+  - "/lib/*.la"
+  - "/lib/pkgconfig"
+  - "/include"
+  - "/lib/cmake"
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+command: rapid-photo-downloader
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+
+  #  For USB, until there is a better way in flatpak
+  - --device=all
+
+  # XDG commons folders
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+
+  # external disk mount
+  - --filesystem=/media
+  - --filesystem=/run/media
+  - --filesystem=/mnt
+
+  # notifications
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+  # file chooser
+  - --talk-name=org.freedesktop.portal.FileChooser
+
+  # App indicator ( not sure if needed)
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.Unity
+  - --talk-name=com.canonical.indicator.application
+  - --talk-name=org.ayatana.indicator.application
+
+  # Not sure if needed ?
+  # - --unset-env=XDG_DATA_HOME
+
+  # QT and perl libs
+  - --env=QT_PLUGIN_PATH=/app/lib64/plugins:/app/lib/plugins
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --env=PERL5LIB=/app/lib/perl5/site_perl/5.36.0/
+
+  # access to udisk
+  - --system-talk-name=org.freedesktop.UDisks2
+
+  # needed for introspection ?
+  - --talk-name=org.freedesktop.DBus.Introspectable.*
+
+  # Needed for gvfs to work and mtp
+  - --talk-name=org.gtk.vfs.*
+  - --filesystem=xdg-run/gvfs
+  - --filesystem=xdg-run/gvfsd
+
+modules:
+  - shared-modules/intltool/intltool-0.51.json
+
+  # enable inspection for gudev
+  - gudev.json
+
+  # for inspection of gobject
+  - python-pygobject.yaml
+
+  # for building
+  - python-hatchling.yaml
+  - python-hatchling-ext.yaml
+
+  # runtime dep
+  - python-arrow.yaml
+  # mega runtime dep
+  - python-runtimedep.yaml
+
+  # for camaera capacity
+  - python-gphoto2.yaml
+
+  # for camaera capacity
+  - python-pyzmq.yaml
```

Comment:
except python dependencies that are autogenerated with flatpak-pip-generator rest should be in the main manifest.

Reply:
They are not autogenrated because the projet  use hatch and the thereis no longer file to use with pip to generate the dep https://github.com/flatpak/flatpak-builder-tools/issues/419